### PR TITLE
Changed scripts to accept command-line arguments and moved them

### PR DESCRIPTION
**Pull request by willic3**
to new cli directory.
Maybe the directory should be moved down a level.

### CODE REVIEW

File: src/cli/cfm_shp_to_tsurf.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python3
+"""
+Simple script to convert entries from the current CFM shapefile into
+multiple tsurf files.
+"""
+if __name__ == "__main__":
```
Comment (chrisbc) [inline src/cli/cfm_shp_to_tsurf.py:6]: this is valid code, but an unusual use of  `if __name__ == "__main__":`

Take a look at https://stackoverflow.com/questions/419163/what-does-if-name-main-do

You might do it at line 24, and/or you might wrap your main loop & setup in a function , then call function after the `if __name __` test. That's probably a more conventional approach.

**Review by chrisbc — APPROVED** [review]
Hi Charles - I made  a comment about `if __name__` conventions

File: src/cli/cfm_shp_to_tsurf.py (hunk 1 above)
Comment (willic3) [reply src/cli/cfm_shp_to_tsurf.py:6]: Hi Chris, Yes I realise this is a bit unusual, and I do understand the purpose behind the `if __name__ == "__main__":` statement.  I might rearrange things so it makes a bit more sense.

**Comment by willic3** [comment]
Hi Chris, I fixed up the scripts as you suggested and I did a bit of moving around.  Let me know what you think.

**Review by chrisbc — APPROVED** [review]
Nice work @willic3 :+1:

**Pull request merged**